packages/flutter/lib/src/material/navigation_bar.dart

```diff
@@ -615,7 +646,7 @@ class NavigationIndicator extends StatelessWidget {
         final double scale = animation.isDismissed
             ? 0.0
             : Tween<double>(begin: .4, end: 1.0).transform(
-            CurveTween(curve: Curves.easeInOutCubicEmphasized).transform(animation.value));
+            CurveTween(curve: Curves.fastOutSlowIn).transform(animation.value));
```

Comment:
Why change these curves? I believe that the advice about Motion and Material 3 is that we should always use Curves.easeInOutCubicEmphasized curve.

Reply:
After I added the indicator shape ripple and tested the animation progress side by side with Android It felt off to me so I looked into Android implementation to see if it animation curve is the same as in Flutter. After watching it, this looked close to Android. 

See this [source](https://github.com/material-components/material-components-android/blob/d5b41c185ff41243f961784aa5a8382a387aa25a/lib/java/com/google/android/material/navigation/NavigationBarItemView.java#L371)

![Screenshot 2022-11-17 at 20 23 44](https://user-images.githubusercontent.com/48603081/202527315-5cc8a883-098f-42af-8ac8-95a858853bff.png)





If this is a deal breaker, I can revert this change and we can consider it in a future PR.

Reply:
Will revert this for now :)